content/influxdb/v2.0/reference/api/influxdb-1x/_index.md

```diff
@@ -57,6 +57,13 @@ Authorization: Token <token>
 Authorization: Token mYSuP3rs3cREtT0k3N
 ```
 
+##### InfluxQL support
+
+The compatability API supports InfluxQL, with the following caveats:
+
+- The `INTO` clause (e.g. `SELECT ... INTO ...`) is not supported.
```

Comment:
For management commands and moving data, I think we should also mention and link to how it's possible to do the same tasks with the v2 API.

Reply:
@gunnaraasen Would it make sense to link to https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/bucket/delete/ and https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/delete/?